.circleci/config.yml

```diff
@@ -23,35 +23,15 @@ commands:
         failure_message: ':red_circle: M1: A $CIRCLE_JOB job has failed!'
         only_for_branches: master
         webhook: $SLACK_GREENKEEPER_WEBHOOK
-  notify-rc-tag:
-    description: 'Notifies #eng-magento1 when a release candidate is tagged.'
-    parameters:
-      tag:
-        type: string
-        default: ''
-    steps:
-    - slack/notify:
```

Comment:
Maybe we can save this one? It's really readable.

Reply:
Agreed, we should notify slack (unless that is handled elsewhere)

Reply:
It does notify slack. Everything's in the script. Vitaliy simply wants to separate into two steps to make it more clear, which I agree with.

Reply:
Done.